apps/trading/components/market-banner/market-termination-banner.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  ExternalLink,
+  Intent,
+  NotificationBanner,
+} from '@vegaprotocol/ui-toolkit';
+import { useState } from 'react';
+import { useGetTerminationProposals } from '@vegaprotocol/proposals';
+import { useVegaWallet } from '@vegaprotocol/wallet';
+import { t } from '@vegaprotocol/i18n';
+import { useDataProvider } from '@vegaprotocol/data-provider';
+import { positionsDataProvider } from '@vegaprotocol/positions';
+import { formatDateWithLocalTimezone } from '@vegaprotocol/utils';
+import { formatDuration, intervalToDuration } from 'date-fns';
+
+export const MarketTerminationBanner = () => {
+  const [hiddens, setHiddens] = useState<string[]>([]);
+  const { pubKey, pubKeys } = useVegaWallet();
+
+  const partyIds = pubKeys?.map((item) => item.publicKey) || [];
```

Comment:
I think lets keep this much more simple and _not_ fetch positions.

I think it will be important for this banner to show regardless of A) if you have connected your wallet, and B) if your key has positions

Reply:
done